code/sp_module.py

```diff
@@ -8,12 +8,12 @@
 co-pylot API.  
 
 """
+import flux_model
 import os, csv
 import matplotlib.pyplot as plt
 from copylot import CoPylot
 import numpy
 import pandas
```

Comment:
I know you didn't write all of this, but could you rearrange the imports here to PEP 8 standards too?